### ISSUE TITLE
published proposal about polls

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#17]
gather criteria, e.g. 
* how long should a poll last ?
  * based on type of poll
* see hangout 2: a poll shall not acted upon before it ended
* minimum balance/magnitude participation required ? (see also [hangout 3](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2016_08_20.MD): 2:15:39)

and later let community vote on these

mentioned in [hangout 2](https://github.com/Erkan-Yilmaz/Gridcoin-hangout-minutes/blob/master/hangout_2016_08_13.MD): see WuProp section

**Comment by grctest** [comment]
14 days for a whitelist poll (unless there's proven evidence of issues requiring a more prompt removal from the whitelist).
7 days for a casual/fun poll.
21-28 days for an expense?
I was thinking that for technical polls regarding development direction we should make polls which last two or three months to bring as much attention to the subject as possible? Is there a maximum poll length?

**Comment by Erkan-Yilmaz** [comment]
another question about:
whitelist poll:
* e.g. recently, CPDN was [voted out](https://steemit.com/boinc/@erkan/gridcoin-whitelist-should-boinc-project-climateprediction-be-removed-poll-until-januar-8-2017) due to a reason. 

If that reason was fixed, and no other issues would exist. Still 14 days ?
May become less relevant when greylisting comes (task #6)

**Comment by grctest** [comment]
Yeah, I'd say that CPDN's situation where it was not exporting statistics correctly should remain 14 days, less than 14 days for projects that are hacked & distribute malware or something equally bad.

If a project fixes an issue before the end of the whitelist poll, it shouldn't invalidate the poll. They should have to campaign for being whitelisted again, this amount of bureaucracy could be reduced via the theorized grey listing mechanism.

**Comment by Erkan-Yilmaz** [comment]
added above: "minimum balance/magnitude participation required ?"

**Comment by Erkan-Yilmaz** [comment]
Should investors be able to vote on whitelist project removal/addition ?

chat excerpt:

<xXUnRealXx> So I have a seriour question / concern. The POLLs that are created for projects SHOULD NOT allow INVESTORS to VOTE. Theres 2 INVESTORS with Lots of PULL Voting to REMOVE Projects Yet they  do none of the work.

<Gunde> INVESTOR could be users that still does boinc but use an investorwallet.
<Gunde> some user do use both part an split wallets
<Gunde> INVESTOR still have interest to health to gridcoin and to do it better
<Gunde> It could be do to security and be anon to split it
<Gunde> No trackback when vote is one part other is to not get a track of ip or other info as mail

<xXUnRealXx> if so than a 51% attack is eaven easier then they say it is
<Gunde> As cm point out the lost grc that is not staked is biggest issue not those investors that vote today.
<Gunde> There is no trackback as they are infisible today, make POSv3 would be best or even make a sytem to burn after limit as it is so high amount of hidden coins.

**Comment by 47an** [comment]
If no exporting of stats is done or no rac from users it would be de-listed auto do to how NN works. This happend before and Denis/MindeModeling/CPDN have show that it will be out. This could be used as a greyl-ist but today the time is to long to use for this. Solution could be to set rules to NN to keep rac in NN in shorter timerange.

If projects that are hacked & distribute malware we should be able to shutdown reward without asking or use poll for it as it mainly bad and would hurt not only Gridcoin but boinc community it self. Same rules as to community platform or wallet or site we use we would not ask to act to protect these.

**Comment by grctest** [comment]
>Should investors be able to vote on whitelist project removal/addition ?

Yes, I believe that investors should be allowed to vote on project whitelist removal/addition.

* It encourages project owners/volunteers to buy GRC to keep their project voted into the whitelist.
* It increases the security of the whitelist, pools would otherwise be able to vote projects in/out at will.
* If a bad quality BOINC project is voted into the whitelist which results in negative press or damages team Gridcoin's computer environment (malware) then the value of GRC could decrease which would affect an investor.

**Comment by grctest** [comment]
RE: Technical polls --> several month long poll & a minimum vote weight required of 50% before a change can pass? This way, low stake weight participation in a poll does not allow mandate for change to be valid without a majority of the network agreeing with the proposal.

**Comment by barton2526** [comment]
Idea: Foundation Expense polls should require a minimum voting weight of the community in order to be considered binding.  I suggest 10% minimum.  Current active foundation poll activity is:

- 9.4% for Faucet fund
- 3.8% for AdWords Campaign
- 3.2% for Hangout Expenses

**Comment by NeuralMiner** [comment]
Though I agree there should be a minimum, if a minimum were to be set now at 10%... no foundation poll except one (and barely) would've passed in the last 4 months (at least). I'd say if the average was 15% participation in polls and a handful that were below 10%, then absolutely make 10% a minimum, but making 10% a minimum when just one poll in recent memory has even reached that 10% is just asking for trouble; no foundation polls will ever pass. 
You can't force people to vote if they don't want to. 
I don't think the minimum should be set at 15% when the average is 5% or less.

**Comment by Erkan-Yilmaz** [comment]
there is a poll currently about this topic also:
* [Should there be a minimum vote weight required for foundation polls?](https://steemit.com/gridcoin/@erkan/gridcoin-poll-should-there-be-a-minimum-vote-weight-required-for-foundation-polls-vote-until-27th-february) (vote until 27th February)

current standing:
* 53% of the shares want a minimum weight of 10%
* 22% want 15%

**Comment by Erkan-Yilmaz** [comment]
the vote ended: 
* most voted answer is: "yes, 20%" 
* 20% or more vote weight wanted by 56.85% of voter shares

see pic + summary [here](https://steemit.com/gridcoin/@erkan/gridcoin-poll-should-there-be-a-minimum-vote-weight-required-for-foundation-polls-vote-until-27th-february)

**Comment by grctest** [comment]
How should we evaluate the outcome of polls?
* First past the post?
* Proportional representation?

**Comment by Erkan-Yilmaz** [comment]
new, related polls:
* [How should we interpret the outcome of a poll?](https://steemit.com/gridcoin/@erkan/gridcoin-poll-how-should-we-interpret-the-outcome-of-a-poll-vote-until-march-29)
* [What is the minimum vote weight that should be required for mandate behind a poll to be valid?](https://steemit.com/gridcoin/@erkan/gridcoin-poll-what-is-the-minimum-vote-weight-that-should-be-required-for-mandate-behind-a-poll-to-be-valid-vote-until-march-29)

**Comment by startailcoon** [comment]
Regarding whitelist polls and Investors.  Make the polls only Magnitude, not Mag+Balance.

**Comment by Erkan-Yilmaz** [comment]
new, related poll:
* [If A Poll Attempts To Establish A Minimum Vote Weight Requirement For Future Polls , Should The Poll In Question Require An Equal Level Of Vote Weight Participation To Be Considered Legitimate?](https://steemit.com/gridcoin/@erkan/gridcoin-poll-if-a-poll-attempts-to-establish-a-minimum-vote-weight-requirement-for-future-polls-should-the-poll-in-question)

**Comment by mistermarmot** [comment]
Currently there are two investors with something like $50 million in GRC stored and they can vote down any whitelist attempt.  They currently haven't TMK, but that doesn't disallow potential future barrons from blocking all whitelist attempts. 
Is there any project, whatsoever, that has made a large GRC investment to influence GRC activities related to their project?
I also strongly suggest MAG only voting on whitelisting.

**Comment by Erkan-Yilmaz** [comment]
"14 days for a whitelist poll (unless there's proven evidence of issues requiring a more prompt removal from the whitelist)."

Personally, for e.g. [this poll](https://steemit.com/gridcoin/@erkan/gridcoin-poll-should-we-remove-atlas-home-from-the-whitelist-due-to-no-available-work-units-vote-until-april-4), I'd have liked to create it for 7 days, since the reason is obvious (no WUs anymore)

**Comment by startailcoon** [comment]
Personally I think removal should be quicker than including. Removal is usually due to something happening. No WUs, flawed credit system, cheating etc. Of course we need to give users enough time to vote about it, but if the reason is obvious it should be automatic. What are we to do if 'No' get the most votes when its a matter of network security. We need to set up some ground rules regarding this instead so the network can make an automatic consensus without a poll.

**Comment by Scalextrix** [comment]
Because of the way RAC works, it takes time to build new RAC on a new project.  So if we have 1000 RAC on a project to be delisted, that will decay over a number of weeks, to rebuild an equivalent RAC on a new project will take a similar amount of time.  So with ATLAS for example, its no longer active, but its going to take time to rebuild that RAC on LHC@Home.  In these circumstances (where the project is properly managedd and communication is clear) we have a 2 week vote on the dates on which the project will be removed, immediately, 2 weeks or 4 weeks.
If a project admin is lazy, distributing malware etc. the vote should be quick and the removal immediate.

**Comment by Erkan-Yilmaz** [comment]
see also @grcjamezz ' comment about investors in the Pentathlon polls (see comment [here](https://steemit.com/gridcoin/@erkan/gridcoin-poll-how-should-we-do-the-boinc-project-selection-for-the-8th-pentathlon-2017))

**Comment by barton2526** [comment]
Perhaps in the future, for polls such as this [one](https://gridcoinstats.eu/poll/how_should_we_do_the_boinc_project_selection_for_the_8th_pentathlon_2017?) we should simply count the number of user) we should consider @grcjamezz 's suggestion above (see comment [here](https://steemit.com/gridcoin/@erkan/gridcoin-poll-how-should-we-do-the-boinc-project-selection-for-the-8th-pentathlon-2017))

**Comment by XaqFields** [comment]
Just a few ideas: 

1. I think we need to have a published set of rules on how certain types of votes should be cast (IE: whitelist polls) and those rules should be agreed-upon by the community via a vote. Currently it seems we are using suggestions made in this thread to govern polls (or at least this is how @Erkan-Yilmaz is doing it). We should get a firm, voted on set of rules to avoid confusion on what makes a poll invalid. If we don't have that in place, we have no basis for which to challenge the result of a poll. 
2. If a poll is published which does not follow those rules, it should be deleted (if possible). If deletion is not possible, the poll should be allowed to run its course and a second poll should then be launched with the correct parameters. (**A second poll should not be created in parallel as this causes confusion and consternation should the polls produce different results.**) 
3. As it relates to Whitelist polls, I think it should be considered to allow Magnitude to be the driving weight for these polls unless there is a reputational risk related to the project under consideration for addition/removal. There is already a lot of fuss being created in the mining community that they feel they don't have enough weight in many of our polls --- I think at least letting them have the highest weight in Whitelist polls would go a long way toward improving that perception. 
4. As part of our published rules, we should also agree on ways in which a user can have their poll creation privileges revoked. Since these polls are important to the development and reputation of Gridcoin, users who abuse the system and/or create unnecessary/confusing/aggressive polls should rightfully have this privilege revoked.

**Comment by denravonska** [comment]
Filed an issue over at https://github.com/gridcoin/Gridcoin-Research/issues/433. In my opinion all the options bug mag+balance are redundant.

**Comment by Erkan-Yilmaz** [comment]
@XaqFields about [https://github.com/Erkan-Yilmaz/Gridcoin-tasks/issues/17#issuecomment-314790867](this):
#4 those polls you mentioned in chat are not created by me, but I can tell you who created those, or he can step up himself here + explain
#5 sztaki: I have added after your comment a little more info. In general I put less info than too much in polls, b/c as you suggest, it can "influence".

**Comment by denravonska** [comment]
I think all whitelist polls, add or remove, should come with a basic TL;DR. Something like

```
- Reason for removal: Owner is a douche
- SSL: No
- Work units remaining: 90
- Estimated time to completion: 4 years
```

If we can get that info, that is.

**Comment by Erkan-Yilmaz** [comment]
see also:
* [BOINC project whitelist guidelines/checklist ](https://cryptocurrencytalk.com/topic/51660-requesting-peer-review-boinc-project-whitelist-guidelineschecklist/)

**Comment by Erkan-Yilmaz** [comment]
[All whitelist polls in July have ended, but: need to be repeated ?](https://github.com/Erkan-Yilmaz/GRC/issues/11)
* we had for weeks problems with the SB which has caused people fromspecific BOINC projects to have magnitude zero (e.g. currently up to 2 weeks time)

**Comment by Erkan-Yilmaz** [comment]
see also: 
* [adding Sourcefinder (with no WUs) to the whitelist and its effects](https://github.com/Erkan-Yilmaz/GRC/issues/24)
